Changes.md

```diff
@@ -1,6 +1,11 @@
 1.2.0.0ax (relative to 1.2.0.0a2)
 =========
```

Comment:
Funny question that came up in a quick rebase:  what do I do with the type id when I now know that something else has used the next available type id on master?  I've currently left my typeId as is, which currently leaves a gap, which will be filled when it is merged with master.  I guess maybe the most consistent thing to do would be use the next available id, the same id that is already used on master, and assume you will fix it when you merge?

Reply:
> I guess maybe the most consistent thing to do would be use the next available id, the same id that is already used on master, and assume you will fix it when you merge?

Yes, that's what I've been doing so far. My reasoning is that things on 1.1 "happened first" and also get precedence in terms of ABI stability. Ids can change on `main` because that doesn't have any ABI guarantees until it officially becomes 1.2.

Reply:
Fixed.